project_scrum/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Scrum",
+    "summary": "Use Scrum Method to manage your project",
+    "version": "13.0.1.0.0",
+    "category": "Project Management",
+    "author": "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project-agile",
+    "depends": [
+        "base_setup",
+        "project",
```

Comment:
Add file oca_dependencies.txt in the repo:
project https://github.com/OCA/project 13.0